src/unix/index_unix.ml

```diff
@@ -2,6 +2,18 @@ exception RO_not_allowed
 
 let current_version = "00000001"
 
+type stats = { mutable bytes_read : int; mutable nb_reads : int }
```

Comment:
can you do this for writes too? that'll help computing a more precise "write amplification" result in the benchmarks.